BROWSER_README.md

```diff
@@ -0,0 +1,82 @@
+_This is one of the steps in the Truss interview process. If you've
+stumbled upon this repository and are interested in a career with
+Truss, [check out our jobs page](https://truss.works/jobs)._
+
+# Truss Software Engineering Interview
+
+## Introduction and expectations
+
+Hi there! Please complete the problem described below to the best of
+your ability, using the tools you're most comfortable with. Assume
+you're sending your submission in for code review from peers;
+we'll be talking about your submission in your interview in that
+context.
+
+We expect this to take less than 4 hours of actual coding time. Please
+submit a working but incomplete solution instead of spending more time
+on it. We're also aware that getting after-hours coding time can be
+challenging; we'd like a submission within a week and if you need more
+time please let us know.
+
+Approach this solution the way you would a real world problem. Use
+libraries where it makes sense to, and be prepared to explain your
+thought process if you do.
+
+If you have any questions, please contact hiring@truss.works; we're
+happy to help if you're not sure what we're asking for or if you have
+questions.
+
+## How to submit your response
+
+Please send hiring@truss.works a link to a public git repository
+(Github is fine) that contains your code and a README.md that tells us
+how to build and run it. Your code will be run in Firefox, Chrome, or Safari.
+
+## The problem: Display data from an API for human consumption
+
+Please write a static webpage that loads data from `https://swapi.dev/api/planets/` and displays that data
+in an HTML `table`.
+
+Name the entrypoint for your application `index.html`. To test your solution, we will open `index.html` in a browser.
```

Comment:
What if we provide specific info that should encompass all submission-type possibilities? for example:
- If your solution code requires compilation (such as with a tool like webpack or create-react-app), please provide complete instructions, including a package.json file with all dependencies included, for running your code as part of your submission.
- If your solution is in the form of static assets (such as an index.html file) we will run it using a simple no-config HTTP server (https://github.com/vercel/serve)
- If your solution is on a hosted IDE such as Codepen or Glitch, please make sure to provide an evergreen link that does not expire

Reply:
I like this. It covers all of the bases pretty well.